drf_jsonschema_serializer/converters.py

```diff
@@ -203,21 +203,42 @@ def convert(self, field):
                 types.add("number")
             elif choice is None:
                 types.add("null")
+
         if field.allow_null:
             types.add("null")
+
             if enum[0] is not None:
                 enum.insert(0, None)
-                enumNames.insert(0, "")
+                enum_names.insert(0, "")
+
             has_display_names = True
 
         types = sorted(list(types))
-        if len(types) == 1:
-            type = types[0]
-        else:
-            type = types
-        result = {"type": type, "enum": enum}
+        result = {"type": types[0] if len(types) == 1 else types, "enum": enum}
         if has_display_names:
-            result["enumNames"] = enumNames
+            result["enumNames"] = enum_names
+        return result
+
+
+@converter
+class ChoiceField(BaseChoiceFieldConverter):
+    field_class = serializers.ChoiceField
+
+    def convert(self, field):
+        result = super(ChoiceField, self).convert(field)
+        return result
+
+
+@converter
+class MultipleChoiceField(BaseChoiceFieldConverter):
+    field_class = serializers.MultipleChoiceField
+
+    def convert(self, field):
+        super_result = super(MultipleChoiceField, self).convert(field)
+        result = {
+            "items": super_result,
+            "type": "array",
+        }
         return result
```

Comment:
```suggestion
    def convert(self, field):
        super_result = super().convert(field)
        result = {
            "items": super_result,
            "type": "array",
        }
        return result
```
you can use the short version in Python 3